packages/synapse-react-client/src/components/SynapseHomepage.tsx

```diff
@@ -363,67 +434,265 @@ export const SynapseHomepage: React.FunctionComponent<SynapseHomepageProps> = ({
+                standardPlanValue="<=100GB*"
+                managedPlanValue="<=500GB*"
+                dataCoordinationPlanValue="Unlimited"
+              />
+              <SynapsePriceFeatureRow
+                featureDescription="5 years of secure, guaranteed storage in the cloud"
+                managedPlanValue={true}
+                dataCoordinationPlanValue={true}
+              />
+              <SynapsePriceFeatureRow
+                featureDescription="Data storage location"
+                standardPlanValue="US Default***"
+                managedPlanValue="US Default***"
+                dataCoordinationPlanValue="Custom"
+              />
+            </AccordionDetails>
+          </Accordion>
+          <Accordion>
+            <AccordionSummary expandIcon={<ExpandMoreTwoTone />}>
+              <Typography className="PricingFeaturesHeader no-margin">
+                Data Governance Features
+              </Typography>
+            </AccordionSummary>
+            <AccordionDetails sx={{ padding: 0 }}>
+              <SynapsePriceFeatureRow
+                featureDescription="Option to make data available through controlled-access"
+                managedPlanValue={true}
+                dataCoordinationPlanValue={true}
+              />
+            </AccordionDetails>
+          </Accordion>
+          <Accordion>
+            <AccordionSummary expandIcon={<ExpandMoreTwoTone />}>
+              <Typography className="PricingFeaturesHeader no-margin">
+                Data Access Features
+              </Typography>
+            </AccordionSummary>
+            <AccordionDetails sx={{ padding: 0 }}>
+              <SynapsePriceFeatureRow
+                featureDescription="Data access management"
+                standardPlanValue="Self-Service*"
+                managedPlanValue="Self-Service"
+                dataCoordinationPlanValue="Custom"
+              />
+              <SynapsePriceFeatureRow
+                featureDescription="Seamless data access integration with compute environments"
+                dataCoordinationPlanValue={true}
+              />
+              <SynapsePriceFeatureRow
+                featureDescription="Customized data portal"
+                standardPlanValue="Basic"
+                managedPlanValue="Basic"
+                dataCoordinationPlanValue={true}
+              />
+              <SynapsePriceFeatureRow
+                featureDescription="Contract length"
+                managedPlanValue="Length of grant"
+                dataCoordinationPlanValue="Contact Sage**"
+              />
+            </AccordionDetails>
+          </Accordion>
+
+          <Typography
+            className="text-align-left"
+            sx={{
+              fontWeight: 300,
+              fontSize: '12px',
+              lineHeight: '16px',
+              paddingTop: '15px',
+            }}
+          >
+            Sage can create a customized quote for you if, for example, you need
```

Comment:
Still need to figure out these footnote references.  There are asterisks all over the content, but they don't seem to be linked up properly to footnote markers

Reply:
Footnote references linked, and reviewed/approved by Kevin